regression/scripts/plots/.pytest_cache/v/cache/nodeids

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
???

Reply:
I have no clue why these things appear...I attempted a pytest for images but we are going to remove it